scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Policy.scala

```diff
@@ -59,44 +60,44 @@ object Policy {
   def apply(
       endPolicy: End.WithPos,
       noDequeue: Boolean = false
-  )(f: Pf)(implicit line: sourcecode.Line): Policy =
+  )(f: Pf)(implicit fileLine: FileLine): Policy =
     new ClauseImpl(f, endPolicy, noDequeue)
 
   def after(
       token: Token,
       noDequeue: Boolean = false
-  )(f: Pf)(implicit line: sourcecode.Line): Policy =
+  )(f: Pf)(implicit fileLine: FileLine): Policy =
     apply(End.After(token), noDequeue)(f)
 
   def before(
       token: Token,
       noDequeue: Boolean = false
-  )(f: Pf)(implicit line: sourcecode.Line): Policy =
+  )(f: Pf)(implicit fileLine: FileLine): Policy =
     apply(End.Before(token), noDequeue)(f)
 
   def after(trigger: Token, policy: Policy)(implicit
-      line: sourcecode.Line
+      fileLine: FileLine
   ): Policy =
     new Switch(NoPolicy, trigger, policy)
 
   def before(policy: Policy, trigger: Token)(implicit
-      line: sourcecode.Line
+      fileLine: FileLine
   ): Policy =
     new Switch(policy, trigger, NoPolicy)
 
   def on(
       token: Token,
       noDequeue: Boolean = false
-  )(f: Pf)(implicit line: sourcecode.Line): Policy =
+  )(f: Pf)(implicit fileLine: FileLine): Policy =
     apply(End.On(token), noDequeue)(f)
 
-  abstract class Clause(implicit val line: sourcecode.Line) extends Policy {
+  abstract class Clause(implicit val fileLine: FileLine) extends Policy {
     val f: Policy.Pf
     val endPolicy: End.WithPos
 
     override def toString = {
       val noDeqPrefix = if (noDequeue) "!" else ""
-      s"${line.value}$endPolicy${noDeqPrefix}d"
+      s"${fileLine.line.value}$endPolicy${noDeqPrefix}d"
```

Comment:
this is where you use `$fileLine`, no? or is this too verbose? you just wanted it on the split?

Reply:
Added.